crates/router/src/routes/app.rs

```diff
@@ -329,3 +331,21 @@ impl Configs {
             )
     }
 }
+
+pub struct ApiKeys;
+
+#[cfg(any(feature = "olap", feature = "oltp"))]
```

Comment:
All routes won't be for both `olap` and `oltp`.

All Dashboard using will be olap and only the backend api comes in oltp.